nilearn/tests/test_niimg.py

```diff
@@ -16,21 +16,25 @@
 currdir = os.path.dirname(os.path.abspath(__file__))
 
 
+@pytest.fixture
+def img1():
```

Comment:
maybe we could put it in conftest.py or somewhere else where it would be available to other test modules? I think having a tiny image whose values and affine don't matter is a common requirements and gets redefined in several places. a slightly more explicit name might also help reusing it

Reply:
Sure, I think that would be good too but would it be better done in another PR? I think it is worth a discussion first how we want to move forward refactoring and modularizing fixtures. For example, having all fixtures in one file and where, creating new fixtures for test files where we see this repetition, better naming for fixtures, etc. WDYT?

Reply:
sure, that  makes more sense!